etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-volumes.json

```diff
@@ -1055,7 +1055,7 @@
                 "dashLength": 10,
                 "dashes": false,
                 "datasource": "",
-                "description": "The IOPS panel displays volume IOPS based on brick-level read and write operations fetched using “gluster volume profile\".",
+                "description": "The IOPS panel displays aggregated brick level read and write operations collected using “gluster volume profile info” over a period of time. The IOPS value shown only reflects metrics from volumes that have volume profiling enabled. No data will be shown if volume profiling is not enabled on at least 1 volume.",
```

Comment:
Same here as well

Reply:
https://github.com/Tendrl/monitoring-integration/pull/498#discussion_r197356500

Reply:
Probably need to use :
"The IOPS panel displays aggregated brick level read and write operations
collected using “gluster volume profile info” over a period of time.  The IOPS
value shown only reflects metrics from volumes that have volume profiling
enabled.  No data will be shown if at-least one brick is not part of a volume for which the volume profiling is enabled."